### PR TITLE
FEAT/Implemented custom layout for pages not using the global layout

### PR DESCRIPTION
**Pull request by 0xlarmideh**
@brightiortsor review

### CODE REVIEW

**Comment by 0xlarmideh** [comment]
I think we have to create some database at some point to mock how this works. Every category will have a nested object that includes all the products, available items, price and all infos. That way we can display at the bottom a mapped view of a category once clicked. Will look into creating that kind of database with sanity.

File: components/Login/LoginForm.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+} from "@mui/material";
+import Container from "@mui/material/Container";
+import AccountCircleIcon from "@mui/icons-material/AccountCircle";
+import Visibility from '@mui/icons-material/Visibility';
+import VisibilityOff from '@mui/icons-material/VisibilityOff';
+
+const LoginForm = () => {
+
+  const [showPassword, setShowPassword] = React.useState(false);
+  const handleClickShowPassword = () => setShowPassword((show) => !show);
```
Comment (brightiortsor) [inline components/Login/LoginForm.tsx:22]: This function name is too long `handleClickShowPassword`.
`showPassword` or `handlePassword` recommended.

File: components/Login/LoginForm.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  const handleMouseDownPassword = (event: React.MouseEvent<HTMLButtonElement>) => {
```
Comment (brightiortsor) [inline components/Login/LoginForm.tsx:23]: This is also long `handleMouseDownPassword`.
Try to keep your function names short yet explanable.

File: components/Login/Logo.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+const Logo = () => {
+  return (
+    <Container sx={{
+      display: "flex",
+      justifyContent: "center",
+      gap: "10px",
+      alignItems: "center"
+    }}>
+      <WidgetsOutlinedIcon sx={{ color: "black" }} />
```
Comment (brightiortsor) [inline components/Login/Logo.tsx:12]: Colors are defined in /config/colors and you're obviously not importing this "black" from our colors. We need to maintain consistency.

File: components/Login/Logo.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+      <Typography variant="h5">Vita OS</Typography>
```
Comment (brightiortsor) [inline components/Login/Logo.tsx:13]: Is this `Vista OS` or `Vista POS`?

**Review by brightiortsor — APPROVED** [review]
Attach a screenshot(s) of your work next time just the way I did my PR please.

Nice job btw.

**Comment by 0xlarmideh** [comment]
All noted.

**Pull request merged**